src/containers/Review/assignment/AssignmentRows.tsx

```diff
@@ -31,12 +31,15 @@ const AssignmentRows: React.FC<AssignmentRowsProps> = ({
   return (
     <>
       {Object.values(fullStructure.sections).map(({ details: { id, title, code } }) => (
-        <Segment className="stripes" key={id}>
+        <Segment key={id}>
           <Header className="section-title" as="h5" content={title} />
           {Object.entries(assignmentGroupedLevel).map(([level, assignments]) => (
-            <>
+            <div
+              className="flex-column"
+              key={`assignment-group-level-${level}`}
+              // style={{ gap: 10 }}
+            >
               <AssignmentSectionRow
-                key={`assignment-row-section-${code}-level-${level}`}
```

Comment:
Oh, it doesn't need this then? Maybe also not required in `ReviewSectionRow` below?